src/core/hle/service/cfg/cfg.cpp

```diff
@@ -413,6 +416,11 @@ ResultCode FormatConfig() {
     if (!res.IsSuccess() && res.description != ErrorDescription::FS_FileNotFound) {
         return res;
     }
+
+    // Temporary store the current console_id
+    u64 console_id = 0;
```

Comment:
This code shouldn't exist.
Formatting the config should yield you a new friendcodeseed, it shouldn't be carried over.